CHANGES.md

```diff
@@ -1,3 +1,41 @@
+## edge-19.9.4
+
+This edge release introduces experimental support for distributed tracing as
+well as a redesigned sidebar in the Web UI!
+
+Experimental support for distributed tracing means that Linkerd data plane
+proxies can now emit trace spans, allowing you to see the exact amount of time
+spent in the Linkerd proxy for traced requests. The new
+`config.linkerd.io/trace-collector` and
+`config.alpha.linkerd.io/trace-collector-service-account` tracing annotations
+allow specifying which pods should emit trace spans.
+
+The goal of the dashboard's sidebar redesign was to reduce load on Prometheus
+and simplify navigation by providing top-level views centered around namespaces
+and workloads.
+
+* CLI
+  * Introduced a new `--cluster-domain` flag to the `linkerd install` command
+    that allows setting a custom cluster domain (thanks @arminbuerkle!)
+  * Fixed the `linkerd endpoints` command to use the correct Destination API
+    address (thanks @Pothulapati!)
+  * Added `--disable-heartbeat` flag for `linkerd` `install|upgrade` commands
+* Controller
+  * Instrumented the proxy-injector to provide additional metrics about
+    injection (thanks @Pothulapati!)
+  * Added support for `config.linkerd.io/skip-webhook` annotation on namespaces
```

Comment:
The should be `config.linkerd.io/admission-webhooks: disabled` label; not an annotation.